cmd/server/server.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func NewServer() *http.Server {
```

Comment:
**Suggestion:**
It would be a good idea to add a mandatory argument for configurations. This would be something like a new package or file that has an exported function to create configs (a struct) like port, host, etc.

`func NewServer(cfg *Configs) *http.Server`